rules/docker-dhcp-server.mk

```diff
@@ -15,6 +15,7 @@ $(DOCKER_DHCP_SERVER)_LOAD_DOCKERS = $(DOCKER_CONFIG_ENGINE_BULLSEYE)
 
 $(DOCKER_DHCP_SERVER)_INSTALL_DEBS = $(PYTHON3_SWSSCOMMON)
 $(DOCKER_DHCP_SERVER)_PYTHON_WHEELS += $(SONIC_DHCP_SERVER_PY3)
```

Comment:
@yaqiangz should we add this only when DHCP_SERVER is enabled in build config? 

Refer to: https://github.com/sonic-net/sonic-buildimage/blob/fa05bf183acce96538b8bf124664023ee87ffa17/rules/docker-dhcp-relay.mk#L20

Reply:
Closed